release-notes/7.0/preview/api-diff/preview1/Microsoft.NETCore.App/7.0-preview1_System.md

```diff
@@ -0,0 +1,27 @@
+# System
+
+``` diff
+ namespace System {
+     public static class AppContext {
++        public static void SetData(string name, object? data);
+     }
+     public class ArgumentNullException : ArgumentException {
++        public unsafe static void ThrowIfNull(void* argument, string? paramName = null);
```

Comment:
The `string? paramName = null` is missing a `[CallerArgumentExpression("argument")]`... do we not show such attributes in the diff?

Reply:
Apparently not. I opened this bug to track these missing attributes: https://github.com/dotnet/arcade/issues/8516